include/picongpu/particles/atomicPhysics2/interfaceDefintion.hpp

```diff
@@ -0,0 +1,36 @@
+
+#include <pmacc/math/Vector.hpp>
+
+namespace picongpu::particles
+{
+    namespace compileTime = pmacc::math::CT;
+
+    // in-cell position to initialize macro particle to
+    CONST_VECTOR(
+        /*datatype*/ float_X,
+        /*dim*/ 3,
+        /*name*/ InCellOffset,
+        /*x*/ 0.0,
+        /*y*/ 0.0,
+        /*z*/ 0.0);
+
+    // cell in superCell to initialize macro particle to
+    CONST_VECTOR(uint32_t, 3, CellIdx, 0u, 0u, 0u);
```

Comment:
should be `simDim` or is atomic physics assuming always 3 components?

Reply:
fixed